astropy/io/registry/base.py

```diff
@@ -238,9 +255,10 @@ def my_identifier(*args, **kwargs):
         if not (data_format, data_class) in self._identifiers or force:
             self._identifiers[(data_format, data_class)] = identifier
         else:
-            raise IORegistryError("Identifier for format '{}' and class '{}' is "
-                                  'already defined'.format(data_format,
-                                                           data_class.__name__))
+            raise IORegistryError(
+                "Identifier for format '{}' and class '{}' is "
+                'already defined'.format(data_format, data_class.__name__)
```

Comment:
Should we convert this (and the rest of the ones modified) `.format` statement into an f-string?

Reply:
I think so. I wonder why pyupgrade isn't doing this.